server/lib/chat/chat-models.ts

```diff
@@ -381,13 +381,55 @@ export async function updateUserPermissions(
   }
 }
 
+export async function countBannedChannelIdentifiers(
```

Comment:
nit: name is a bit confusing to me because it sounds like the identifiers are for the channel, not the user. Maybe `countBannedIdentifiersForChannel`?